package.json

```diff
@@ -36,6 +33,11 @@
     "type": "git",
     "url": "git+https://github.com/drone/ff-javascript-client-sdk.git"
   },
+  "dependencies": {
+    "jwt-decode": "*",
+    "mitt": "*",
+    "event-source-polyfill": "*"
```

Comment:
Any specific reason to polyfill `EventSource`? It looks like it has wider support than `globalThis` and we use that freely. Just curious